encoding/yaml/devDeps.ts

```diff
@@ -0,0 +1,3 @@
+import * as asserts from "../testing/asserts.ts";
```

Comment:
Is this file needed?

Reply:
Not anymore, I forgot to remove it after adding the module from 3rd party to std.

Removed.